.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: cpp
+
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      compiler: gcc
+    - os: osx
+      osx_image: xcode9.4
```

Comment:
We support xcode 7.3.1 and greater, so can we add another matrix row with xcode7.3?

Reply:
first you should focus on `xcode9.4` and investigate why some tests are not passing on macOS (race cond? performance?), then add xcode7.3 if you want.

note: since you are in `abseil/` instead of `google/` you have currently only one osx worker...